service/src/main/java/uk/gov/hmcts/reform/fpl/validators/HasTelephoneValidator.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.reform.fpl.validators;
+
+import uk.gov.hmcts.reform.fpl.model.Applicant;
+import uk.gov.hmcts.reform.fpl.validators.interfaces.HasTelephone;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class HasTelephoneValidator implements ConstraintValidator<HasTelephone, Applicant> {
+    @Override
+    public void initialize(HasTelephone constraintAnnotation) {
+
+    }
+
+    @Override
+    public boolean isValid(Applicant applicant, ConstraintValidatorContext constraintValidatorContext) {
+
+        if (applicant.getTelephone() != null || applicant.getMobile() != null) {
```

Comment:
Thing about how to improve in this instance I believe it would be simplified to return isNotBlank(applicant.getTelephone()) || isNotBlank(applicant.getMobile()) (if one of them is enough to pass validation)